sdl_viewer/shaders/outlinedBox.vs

```diff
@@ -0,0 +1,9 @@
+#version 330 core
+layout (location = 0) in vec3 aPos;
```

Comment:
same comment about consistency with existing buffers for naming. Also here I do not care, though I think we are consistent with the naming of the web_viewer, so it is less work if you adapt to whats there.